code/modules/projectiles/projectile/magic.dm

```diff
@@ -374,3 +374,12 @@
 	flag = "magic"
 	dismemberment = 50
 	nodamage = 0
+
+/obj/item/projectile/magic/arcane_barrage
+	name = "arcane bolt"
+	icon_state = "arcane_barrage"
+	damage = 20
+	damage_type = BURN
+	nodamage = 0
+	armour_penetration = 100
```

Comment:
I'm just not sure about this value. I don't know if it's too strong. Would like suggestions for it.